bin/install_swiftpm_dependencies

```diff
@@ -1,9 +1,67 @@
 #!/bin/bash -eu
 
+CUSTOM_PACKAGE_RESOLVED_PATH=${1-}
+BUILD_TYPE=${2-}
+SPM_CACHE_LOCATION="$HOME/spm"
```

Comment:
There exists a `$HOME/.swiftpm/cache` folder on my Mac that was probably created by SPM itself at some point.

I wonder if we shouldn't use that folder (for conventions) as opposed to `~/spm` here?

Reply:
Ah, good point! Jeremy and I had discussed using the cache that's within DerivedData, but I wasn't able to make that work because there's not a consistent path name within DerivedData. But it does work great with making the cache location in the script point to `$HOME/Library/Caches/org.swift.swiftpm`. And that has the added benefit of client repos not needing to specify a specific SPM cache folder in the Fastfile. Package resolution will pull from that `org.swift.swiftpm` folder automatically when resolving the packages. 

Added in https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/43/commits/44785e46a581287c90a5f3d2f021440a33cec2bf